uwblueprint/abtc
# Adding accept reject functionality to admin dashboard

## Notion ticket link

[Add the Reject/Accept functionality to the admin dashboard](https://www.notion.so/uwblueprintexecs/Add-the-Reject-Accept-functionality-to-the-admin-dashboard-3110af6713d94c049848e00f37980653?pvs=4)



## Implementation description
* After selecting any number of platformSignupRequests from the admin dashboard, the admin can press either the checkmark to accept
* Iterate through each of the platformSignupRequests and send a request to delete platformsignup and change status of user


## Steps to test
1. Make a dummy account (or multiple)
2. Check that it is in the list of platform signups
3. Check off dummy account(s)
4. Click accept
5. Verify it is off the screen and database



## What should reviewers focus on?
* The way I structured the additional backend


## Checklist
